### DISCUSSION TITLE
Starting multiple instances of an ios or android app

### DISCUSSION THREAD

**Discussion by sakon54321** [quasarframework/quasar#9241] [CLI - Capacitor / Cordova modes]
Hi, 

I'm trying to start more than one instance of my mobile app so I can test the web-sockets functionality. Basically I want to log in as two different users at the same time and see the updates propagating to both as they happen. 

But when I run: 

quasar dev -m cordova -T ios

from different console windows it just restarts the first instance of the simulator. 

I also tried to start one ios and one android version using:

quasar dev -m cordova -T android
quasar dev -m cordova -T ios

but only one will connect to my local database. 

Anyone know how i can achieve this? 

Thanks in advance.

**Comment by ontwikkelfabriek** [comment]
You can only run one `dev` process from the same source as far as I know. 

What you can do is instead:
1) start ios with `quasar dev cordova -T ios`
2) build the android version with `quasar build -m cordova -T android`
3) go into the `src-cordova` folder and do `cordova run android` 

Another possible solution if you want to have both versions running with `quasar dev` is to clone the project into another folder. Then just simple run `quasar dev` in the other folder to start the other device. ( Quasar dev server will run on another port automatically if 8080 is already taken)

**Reply by sakon54321** [reply]
Thanks for your help. I tried this out and it sort of works. 

I ended up doing:
1. Build the ios app with ```quasar build -m cordova -T ios```
2. go to ```src-cordova``` and run ```cordova run ios```
3. run ```quasar dev cordova -T ios```

Although it opened up the dev app in a safari window instead of a simulator and only one seems to be listening to the websocket port 6001. The android app seems to refuse to log in completely so I'm not sure what's happening there.
